docks-ui.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
```

Comment:
What is this file for?

Reply:
I have no idea, I thought you added this xDD I'll remove it.